### ISSUE TITLE
TypeError RequestList.py

### ISSUE THREAD

**Issue by Helpi90** [odwdinc/Python-SimConnect#47]
Hey guys,

i have a problem with the RequestList.py. I cloned this repo and tried to execute local_example.py and glass_server.py. 

Could someone help me?

Python version: 3.8.6 64Bit

Log:
```
python glass_server.py
Traceback (most recent call last):
  File "glass_server.py", line 14, in <module>
    aq = AircraftRequests(sm, _time=10)
  File "C:\Users\Michael\Desktop\Python-SimConnect\SimConnect\RequestList.py", line 192, in __init__
    self.EngineData = self.__AircraftEngineData(_sm, _time, _attemps)
TypeError: __init__() takes from 2 to 3 positional arguments but 4 were given

python local_example.py
INFO:__main__:START
DEBUG:SimConnect.SimConnect:Connected to Flight Simulator!
INFO:SimConnect.SimConnect:SIM OPEN
Traceback (most recent call last):
  File "local_example.py", line 15, in <module>
    aq = AircraftRequests(sm)
  File "C:\Users\Michael\Desktop\Python-SimConnect\SimConnect\RequestList.py", line 192, in __init__
    self.EngineData = self.__AircraftEngineData(_sm, _time, _attemps)
TypeError: __init__() takes from 2 to 3 positional arguments but 4 were given
```

**Comment by odwdinc** [comment]
There’s a bug, give me a few hours and all push a fix.
Sorry for that, miss a line on the last update.

**Comment by odwdinc** [comment]
@Helpi90 Fixed in 0.4.13
The updated took a bit to download.
